app/src/main/java/com/example/taskwise_eventmaster/data/persistance/daos/ThumbnailDao.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.taskwise_eventmaster.data.persistance.daos
+
+import androidx.room.Dao
+import androidx.room.Query
+import androidx.room.Upsert
+import com.example.taskwise_eventmaster.data.persistance.model.Thumbnail
+@Dao
+interface ThumbnailDao {
+    @Upsert
+    suspend fun upsertThumbnail(thumbnail: Thumbnail)
+
+    @Query("SELECT * FROM Thumbnail WHERE eventId =:eventId")
+    fun getAllThumbnailsForThisEvent(eventId: Int): List<Thumbnail>
```

Comment:
I would advise avoiding using pronouns in the naming as it is discouraged by Clean Code conventions. The goal is to use as descriptive names as needed but avoid redundancy and use only words that add meaning